packages/app/src/themes/dark_theme.tsx

```diff
@@ -1,5 +1,5 @@
 import { createTheme, SimpleThemeOptions } from '@backstage/theme';
-import { gold, common, myYellow as yellow } from "./colorTypes";
+import { blue, gray, green, gold, red, common } from "./colorTypes";
```

Comment:
```suggestion
 /* tslint:disable */
import { blue, gray, green, gold, red, common } from "./colorTypes";
 /* tslint:enable */
```

If they are needed we can disable the check for this one line like this.